paper.tex

```diff
@@ -93,12 +93,14 @@
 We propose the Demes data model and file format,
 built on widely used technologies,
 to alleviate these issues. Demes provides a well-defined and unambiguous
-graphical model of populations and their properties that is straightforward to
+model of populations and their properties that is straightforward to
```

Comment:
I think the word "graphical" here might be too easily misinterpreted as meaning "visual/diagrammatic" instead of meaning a network. We don't mention a demes model as being a directed graph anywhere else, so we may as well delete this word.